atlasdb-workload-server-api/src/main/java/com/palantir/atlasdb/workload/store/ValidationStore.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * (c) Copyright 2023 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.workload.store;
+
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Store used for replaying witnessed transactions, and ultimately being used as the 'source-of-truth' when validating
+ * our {@link TransactionStore}.
+ */
+public interface ValidationStore {
+
+    /** Return all cells and values for all tables. */
+    Map<TableWorkloadCell, Integer> values();
```

Comment:
lol so this may totally OOM us long-term, but for the short/medium term this is OK. In any case, the longer term solution here is we should just have an `Iterator` instead of a map, which then under-the-hood could query sqlite (or some other off-heap storage mechanism).